apps/src/studio/Sprite.js

```diff
@@ -5,8 +5,7 @@ var spriteActions = require('./spriteActions');
 var Direction = constants.Direction;
 var Emotions = constants.Emotions;
 var NextTurn = constants.NextTurn;
-var utils = require('../utils');
-var _ = utils.getLodash();
```

Comment:
Ah, see there were a bunch. Assuming the rest are accurate / would get caught as unknown references during lint or build.